### ISSUE TITLE
Trying to launch pack breaks MC launcher

### ISSUE THREAD

**Issue by jontaylor224** [AllTheMods/ATM-4#127]
Using Twich Launcher Jar launcher on Windows 10 latest version with latest Java version.
Other modpacks launch just fine, tried to launch ATM-4 and the launcher stops with "Loading..." in the play button, and "Loading verisons..." under the user name.  
After this no other modpacks will launch, always giving the same errors in the logs.

The Launcher log:  

[22:49:33 INFO]: Minecraft Launcher 1.6.91 (through bootstrap 100) started on windows...
[22:49:33 INFO]: Current time is Jan 19, 2020 10:49:33 PM
[22:49:33 INFO]: System.getProperty('os.name') == 'Windows 10'
[22:49:33 INFO]: System.getProperty('os.version') == '10.0'
[22:49:33 INFO]: System.getProperty('os.arch') == 'amd64'
[22:49:33 INFO]: System.getProperty('java.version') == '1.8.0_241'
[22:49:33 INFO]: System.getProperty('java.vendor') == 'Oracle Corporation'
[22:49:33 INFO]: System.getProperty('sun.arch.data.model') == '64'
[22:49:33 INFO]: proxy == DIRECT
[22:49:33 INFO]: JFX is already initialized
[22:49:33 INFO]: Refreshing local version list...
[22:49:33 INFO]: Refreshing remote version list...
[22:49:34 INFO]: Refresh complete.
[22:49:34 INFO]: Installed CompleteVersion{id='1.12.2', updatedTime=Mon Sep 18 03:39:46 CDT 2017, releasedTime=Mon Sep 18 03:39:46 CDT 2017, type=RELEASE, libraries=[Library{name='com.mojang:patchy:1.1', rules=null, natives=null, extract=null}, Library{name='oshi-project:oshi-core:1.1', rules=null, natives=null, extract=null}, Library{name='net.java.dev.jna:jna:4.4.0', rules=null, natives=null, extract=null}, Library{name='net.java.dev.jna:platform:3.4.0', rules=null, natives=null, extract=null}, Library{name='com.ibm.icu:icu4j-core-mojang:51.2', rules=null, natives=null, extract=null}, Library{name='net.sf.jopt-simple:jopt-simple:5.0.3', rules=null, natives=null, extract=null}, Library{name='com.paulscode:codecjorbis:20101023', rules=null, natives=null, extract=null}, Library{name='com.paulscode:codecwav:20101023', rules=null, natives=null, extract=null}, Library{name='com.paulscode:libraryjavasound:20101123', rules=null, natives=null, extract=null}, Library{name='com.paulscode:librarylwjglopenal:20100824', rules=null, natives=null, extract=null}, Library{name='com.paulscode:soundsystem:20120107', rules=null, natives=null, extract=null}, Library{name='io.netty:netty-all:4.1.9.Final', rules=null, natives=null, extract=null}, Library{name='com.google.guava:guava:21.0', rules=null, natives=null, extract=null}, Library{name='org.apache.commons:commons-lang3:3.5', rules=null, natives=null, extract=null}, Library{name='commons-io:commons-io:2.5', rules=null, natives=null, extract=null}, Library{name='commons-codec:commons-codec:1.10', rules=null, natives=null, extract=null}, Library{name='net.java.jinput:jinput:2.0.5', rules=null, natives=null, extract=null}, Library{name='net.java.jutils:jutils:1.0.0', rules=null, natives=null, extract=null}, Library{name='com.google.code.gson:gson:2.8.0', rules=null, natives=null, extract=null}, Library{name='com.mojang:authlib:1.5.25', rules=null, natives=null, extract=null}, Library{name='com.mojang:realms:1.10.22', rules=null, natives=null, extract=null}, Library{name='org.apache.commons:commons-compress:1.8.1', rules=null, natives=null, extract=null}, Library{name='org.apache.httpcomponents:httpclient:4.3.3', rules=null, natives=null, extract=null}, Library{name='commons-logging:commons-logging:1.1.3', rules=null, natives=null, extract=null}, Library{name='org.apache.httpcomponents:httpcore:4.3.2', rules=null, natives=null, extract=null}, Library{name='it.unimi.dsi:fastutil:7.1.0', rules=null, natives=null, extract=null}, Library{name='org.apache.logging.log4j:log4j-api:2.8.1', rules=null, natives=null, extract=null}, Library{name='org.apache.logging.log4j:log4j-core:2.8.1', rules=null, natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl:2.9.4-nightly-20150209', rules=[Rule{action=ALLOW, os=null, features=null}, Rule{action=DISALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl_util:2.9.4-nightly-20150209', rules=[Rule{action=ALLOW, os=null, features=null}, Rule{action=DISALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl-platform:2.9.4-nightly-20150209', rules=[Rule{action=ALLOW, os=null, features=null}, Rule{action=DISALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives={LINUX=natives-linux, OSX=natives-osx, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@382968ec}, Library{name='org.lwjgl.lwjgl:lwjgl:2.9.2-nightly-20140822', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl_util:2.9.2-nightly-20140822', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl-platform:2.9.2-nightly-20140822', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives={LINUX=natives-linux, OSX=natives-osx, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@60ebc344}, Library{name='net.java.jinput:jinput-platform:2.0.5', rules=null, natives={LINUX=natives-linux, OSX=natives-osx, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@332d5e58}, Library{name='com.mojang:text2speech:1.10.3', rules=null, natives=null, extract=null}, Library{name='com.mojang:text2speech:1.10.3', rules=null, natives={LINUX=natives-linux, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@31de2ef7}, Library{name='ca.weblite:java-objc-bridge:1.0.0', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives={OSX=natives-osx}, extract=com.mojang.launcher.versions.ExtractRules@42434e28}, Library{name='ca.weblite:java-objc-bridge:1.0.0', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}], mainClass='net.minecraft.client.main.Main', jar='null', minimumLauncherVersion=18}
[22:49:34 INFO]: Installed CompleteVersion{id='1.12.2', updatedTime=Mon Sep 18 03:39:46 CDT 2017, releasedTime=Mon Sep 18 03:39:46 CDT 2017, type=RELEASE, libraries=[Library{name='com.mojang:patchy:1.1', rules=null, natives=null, extract=null}, Library{name='oshi-project:oshi-core:1.1', rules=null, natives=null, extract=null}, Library{name='net.java.dev.jna:jna:4.4.0', rules=null, natives=null, extract=null}, Library{name='net.java.dev.jna:platform:3.4.0', rules=null, natives=null, extract=null}, Library{name='com.ibm.icu:icu4j-core-mojang:51.2', rules=null, natives=null, extract=null}, Library{name='net.sf.jopt-simple:jopt-simple:5.0.3', rules=null, natives=null, extract=null}, Library{name='com.paulscode:codecjorbis:20101023', rules=null, natives=null, extract=null}, Library{name='com.paulscode:codecwav:20101023', rules=null, natives=null, extract=null}, Library{name='com.paulscode:libraryjavasound:20101123', rules=null, natives=null, extract=null}, Library{name='com.paulscode:librarylwjglopenal:20100824', rules=null, natives=null, extract=null}, Library{name='com.paulscode:soundsystem:20120107', rules=null, natives=null, extract=null}, Library{name='io.netty:netty-all:4.1.9.Final', rules=null, natives=null, extract=null}, Library{name='com.google.guava:guava:21.0', rules=null, natives=null, extract=null}, Library{name='org.apache.commons:commons-lang3:3.5', rules=null, natives=null, extract=null}, Library{name='commons-io:commons-io:2.5', rules=null, natives=null, extract=null}, Library{name='commons-codec:commons-codec:1.10', rules=null, natives=null, extract=null}, Library{name='net.java.jinput:jinput:2.0.5', rules=null, natives=null, extract=null}, Library{name='net.java.jutils:jutils:1.0.0', rules=null, natives=null, extract=null}, Library{name='com.google.code.gson:gson:2.8.0', rules=null, natives=null, extract=null}, Library{name='com.mojang:authlib:1.5.25', rules=null, natives=null, extract=null}, Library{name='com.mojang:realms:1.10.22', rules=null, natives=null, extract=null}, Library{name='org.apache.commons:commons-compress:1.8.1', rules=null, natives=null, extract=null}, Library{name='org.apache.httpcomponents:httpclient:4.3.3', rules=null, natives=null, extract=null}, Library{name='commons-logging:commons-logging:1.1.3', rules=null, natives=null, extract=null}, Library{name='org.apache.httpcomponents:httpcore:4.3.2', rules=null, natives=null, extract=null}, Library{name='it.unimi.dsi:fastutil:7.1.0', rules=null, natives=null, extract=null}, Library{name='org.apache.logging.log4j:log4j-api:2.8.1', rules=null, natives=null, extract=null}, Library{name='org.apache.logging.log4j:log4j-core:2.8.1', rules=null, natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl:2.9.4-nightly-20150209', rules=[Rule{action=ALLOW, os=null, features=null}, Rule{action=DISALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl_util:2.9.4-nightly-20150209', rules=[Rule{action=ALLOW, os=null, features=null}, Rule{action=DISALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl-platform:2.9.4-nightly-20150209', rules=[Rule{action=ALLOW, os=null, features=null}, Rule{action=DISALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives={LINUX=natives-linux, OSX=natives-osx, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@7937d909}, Library{name='org.lwjgl.lwjgl:lwjgl:2.9.2-nightly-20140822', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl_util:2.9.2-nightly-20140822', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}, Library{name='org.lwjgl.lwjgl:lwjgl-platform:2.9.2-nightly-20140822', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives={LINUX=natives-linux, OSX=natives-osx, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@f74709e}, Library{name='net.java.jinput:jinput-platform:2.0.5', rules=null, natives={LINUX=natives-linux, OSX=natives-osx, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@251ad292}, Library{name='com.mojang:text2speech:1.10.3', rules=null, natives=null, extract=null}, Library{name='com.mojang:text2speech:1.10.3', rules=null, natives={LINUX=natives-linux, WINDOWS=natives-windows}, extract=com.mojang.launcher.versions.ExtractRules@a8eebd6}, Library{name='ca.weblite:java-objc-bridge:1.0.0', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives={OSX=natives-osx}, extract=com.mojang.launcher.versions.ExtractRules@4d36841f}, Library{name='ca.weblite:java-objc-bridge:1.0.0', rules=[Rule{action=ALLOW, os=OSRestriction{name=OSX, version='null', arch='null'}, features=null}], natives=null, extract=null}], mainClass='net.minecraft.client.main.Main', jar='null', minimumLauncherVersion=18}
[22:49:35 FATAL]: Unhandled exception in thread Thread[AWT-EventQueue-0,6,main]
com.google.gson.JsonSyntaxException: duplicate key: null
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:190) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:93) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:172) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:803) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:768) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:717) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:689) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.RemoteVersionList.getCompleteVersion(RemoteVersionList.java:32) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getLatestCompleteVersion(MinecraftVersionManager.java:213) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:268) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:255) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersionSyncInfo(MinecraftVersionManager.java:171) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersions(MinecraftVersionManager.java:105) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayerInfoPanel.checkState(PlayerInfoPanel.java:96) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayerInfoPanel$2.run(PlayerInfoPanel.java:78) ~[launcher.jar:1.6.91]
	at java.awt.event.InvocationEvent.dispatch(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue.access$500(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue$3.run(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue$3.run(Unknown Source) ~[?:1.8.0_241]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_241]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue.dispatchEvent(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_241]
[22:49:35 ERROR]: Unexpected exception refreshing profile list
com.google.gson.JsonSyntaxException: duplicate key: null
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:190) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:93) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:172) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:803) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:768) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:717) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:689) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.RemoteVersionList.getCompleteVersion(RemoteVersionList.java:32) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getLatestCompleteVersion(MinecraftVersionManager.java:213) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:268) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:255) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersionSyncInfo(MinecraftVersionManager.java:171) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersions(MinecraftVersionManager.java:105) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.game.GameLaunchDispatcher.getStatus(GameLaunchDispatcher.java:35) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayButtonPanel.checkState(PlayButtonPanel.java:94) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayButtonPanel.onProfilesRefreshed(PlayButtonPanel.java:90) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.profile.ProfileManager.fireRefreshEvent(ProfileManager.java:100) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.profile.ProfileManager.loadProfiles(ProfileManager.java:88) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.Launcher$2.run(Launcher.java:183) [launcher.jar:1.6.91]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_241]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
[22:49:35 INFO]: Refreshing auth...
[22:49:35 INFO]: Logging in with access token
[22:49:35 FATAL]: Unhandled exception in thread Thread[AWT-EventQueue-0,6,main]
com.google.gson.JsonSyntaxException: duplicate key: null
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:190) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:93) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:172) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:803) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:768) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:717) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:689) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.RemoteVersionList.getCompleteVersion(RemoteVersionList.java:32) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getLatestCompleteVersion(MinecraftVersionManager.java:213) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:268) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:255) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersionSyncInfo(MinecraftVersionManager.java:171) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersions(MinecraftVersionManager.java:105) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayerInfoPanel.checkState(PlayerInfoPanel.java:96) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayerInfoPanel$2.run(PlayerInfoPanel.java:78) ~[launcher.jar:1.6.91]
	at java.awt.event.InvocationEvent.dispatch(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue.access$500(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue$3.run(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue$3.run(Unknown Source) ~[?:1.8.0_241]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_241]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventQueue.dispatchEvent(Unknown Source) ~[?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_241]
	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_241]
[22:49:35 ERROR]: Unhandled exception in executor com.mojang.launcher.updater.ExceptionalThreadPoolExecutor$ExceptionalFutureTask@1f0bf473
java.util.concurrent.ExecutionException: com.google.gson.JsonSyntaxException: duplicate key: null
	at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
	at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
	at com.mojang.launcher.updater.ExceptionalThreadPoolExecutor$ExceptionalFutureTask.done(ExceptionalThreadPoolExecutor.java:58) [launcher.jar:1.6.91]
	at java.util.concurrent.FutureTask.finishCompletion(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.FutureTask.setException(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_241]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_241]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: com.google.gson.JsonSyntaxException: duplicate key: null
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:190) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:93) ~[launcher.jar:1.6.91]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:172) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:803) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:768) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:717) ~[launcher.jar:1.6.91]
	at com.google.gson.Gson.fromJson(Gson.java:689) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.RemoteVersionList.getCompleteVersion(RemoteVersionList.java:32) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getLatestCompleteVersion(MinecraftVersionManager.java:213) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:268) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.CompleteMinecraftVersion.resolve(CompleteMinecraftVersion.java:255) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersionSyncInfo(MinecraftVersionManager.java:171) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.updater.MinecraftVersionManager.getVersions(MinecraftVersionManager.java:105) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.game.GameLaunchDispatcher.getStatus(GameLaunchDispatcher.java:35) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayButtonPanel.checkState(PlayButtonPanel.java:94) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.ui.bottombar.PlayButtonPanel.onProfilesRefreshed(PlayButtonPanel.java:90) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.profile.ProfileManager.fireRefreshEvent(ProfileManager.java:100) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.Launcher.ensureLoggedIn(Launcher.java:265) ~[launcher.jar:1.6.91]
	at net.minecraft.launcher.Launcher$2.run(Launcher.java:213) ~[launcher.jar:1.6.91]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_241]
	... 4 more

**Comment by Eld0r** [comment]
I can confirm this behavior. For me the same happened as for you:
```log
[22:49:35 ERROR]: Unexpected exception refreshing profile list
com.google.gson.JsonSyntaxException: duplicate key: null
(...)
[22:49:35 ERROR]: Unhandled exception in executor com.mojang.launcher.updater.ExceptionalThreadPoolExecutor$ExceptionalFutureTask@1f0bf473
java.util.concurrent.ExecutionException: com.google.gson.JsonSyntaxException: duplicate key: null
```
Since I tried to start atm4 0.3.1 I must use the "original launcher" setting in twitch to launch any mod.
_To use new Java with this launcher delete everything exept ".version" file of the included lib (java_x64 - they are from v8 u51) with the files of your latest java runtime in program files (e.g. v8 u241). the version file must stay untuched or else the intelligent twitch launcher try to redownload this old java version (fails and abort startup of original launcher)._

**Comment by jontaylor224** [comment]
After more research this is actually a known bug with the 1.6.91 (legacy) launcher.  https://bugs.mojang.com/browse/MCL-11993
The fix is to download the new 1.6.93 launcher and replace it in your Twitch launcher directory.
There is a link in the bug report on the mojang site for the updated launcher.
